db/config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineDb, defineTable, column } from 'astro:db';
+
+export const Subscribers = defineTable({
+  columns: {
+    id: column.number({ primaryKey: true }),
+    external_id: column.text(),
+    email: column.text(),
+    subscription_status: column.text(),
+    subscription_tier: column.text(),
+  },
+});
+
+export const BlorboEntries = defineTable({
+  columns: {
+    id: column.number({ primaryKey: true }),
+    supporter_id: column.number(),
```

Comment:
You should look into adding "unique" constraints and add one here, also same for "email" and "externalId" in subscribers

Reply:
I've added 'unique' constraints to BlorboEntries id and Subscribers id, email and externalId